advisories/github-reviewed/2023/04/GHSA-q2qj-628g-vhfw/GHSA-q2qj-628g-vhfw.json

```diff
@@ -33,6 +33,44 @@
           ]
         }
       ]
+    },
+    {
+      "package": {
+        "ecosystem": "Packagist",
+        "name": "slim/psr7"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.4.1"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "Packagist",
+        "name": "slim/psr7"
+      },
+      "ranges": [
+        {
+          "type": "ECOSYSTEM",
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.5.1"
```

Comment:
Those ranges look broken to me. A `<1.5.0` range would mark 1.4.1 as impacted, while the fix has been backported. The update range cannot be `<1.4.1, <1.5.1, <1.6.1`. It needs to be `<1.4.1, >=1.5.0 <1.5.1, >=1.6.0 <1.6.1`

Reply:
Oh, interesting. I didn't realise that you can't just list the fixed versions.

Reply:
The worst is that I had filled the form with the minimum version and it got dropped
I can not even update this PR :/

But for anyone with power access, here is the diff
```diff
diff --git a/advisories/github-reviewed/2023/04/GHSA-q2qj-628g-vhfw/GHSA-q2qj-628g-vhfw.json b/advisories/github-reviewed/2023/04/GHSA-q2qj-628g-vhfw/GHSA-q2qj-628g-vhfw.json
index 121d3163b5..4fb713dfa3 100644
--- a/advisories/github-reviewed/2023/04/GHSA-q2qj-628g-vhfw/GHSA-q2qj-628g-vhfw.json
+++ b/advisories/github-reviewed/2023/04/GHSA-q2qj-628g-vhfw/GHSA-q2qj-628g-vhfw.json
@@ -25,7 +25,7 @@
           "type": "ECOSYSTEM",
           "events": [
             {
-              "introduced": "0"
+              "introduced": "1.6.0"
             },
             {
               "fixed": "1.6.1"
@@ -63,7 +63,7 @@
           "type": "ECOSYSTEM",
           "events": [
             {
-              "introduced": "0"
+              "introduced": "1.5.0"
             },
             {
               "fixed": "1.5.1"
@@ -112,4 +112,4 @@
     "github_reviewed_at": "2023-04-18T22:20:42Z",
     "nvd_published_at": null
   }
```